Storage/src/Connection/Rest.php

```diff
@@ -44,8 +44,13 @@ class Rest implements ConnectionInterface
     use UriTrait;
 
     const BASE_URI = 'https://storage.googleapis.com/storage/v1/';
-    const UPLOAD_URI = 'https://storage.googleapis.com/upload/storage/v1/b/{bucket}/o{?query*}';
-    const DOWNLOAD_URI = 'https://storage.googleapis.com/storage/v1/b/{bucket}/o/{object}{?query*}';
+    const UPLOAD_PATH = 'b/{bucket}/o{?query*}';
```

Comment:
Simply removing `UPLOAD_URI` and `DOWNLOAD_URI` could break users if they're using those constants, since we don't have constant visibility. Could you maintain `UPLOAD_URI` and `DOWNLOAD_URI` and mark them as `@deprecated`, similar to the BigQuery change?

Also, note that `UPLOAD_URI` has a different base path than other calls. `/upload/storage/v1` as opposed to `/storage/v1`.